test/tests.zig

```diff
@@ -1583,6 +1594,9 @@ pub fn addModuleTests(b: *std.Build, options: ModuleTestOptions) *Step {
             run.setName(b.fmt("run test {s}", .{qualified_name}));
 
             step.dependOn(&run.step);
+        } else if (target.cpu.arch == .spirv64) {
```

Comment:
```suggestion
        } else if (target.cpu.arch.isSpirV()) {
```

To cover `spirv` and `spirv32` too.